### PR TITLE
added highnoon

### PR DESCRIPTION
**Pull request by step1041**
I did a thing. 

!highnoon returns "Its High Noon" if it is 12:00 PM, else, returns the time.

### CODE REVIEW

File: lib/commands/highnoon.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+const Moment = require('../../node_modules/moment');
+
+
+module.exports = new Clapp.Command(
+  {
+    name: "highnoon",
+    desc: "What time is it?",
+    fn: (argv, context) => {
+      //check for highnoon using Momentjs
+      if(Moment().format('Hm') == "1700") {
```
Comment (SpyMaster356) [inline lib/commands/highnoon.js]: 1700? isn't that 5pm?

File: data/changelog.js [hunk 2]
```diff
@@ -21,6 +21,10 @@ module.exports = [
       msg: 'Boop!'
     },
     {
+      type: 'New command: !highoon',
```
Comment (SpyMaster356) [inline data/changelog.js]: this should be at the top in a new array. I need to update how this changelog works

**Comment by step1041** [comment]
Yeah it is. Derp. I did this at 5 and used it for the testing. I also
noticed a typo in the changelog. 2 second fix.

File: lib/commands/highnoon.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+const Clapp = require('../modules/clapp-discord');
+const Moment = require('../../node_modules/moment');
+
+
+module.exports = new Clapp.Command(
+  {
+    name: "highnoon",
```
Comment (SpyMaster356) [inline lib/commands/highnoon.js]: !time may be a better command name

File: lib/commands/highnoon.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    desc: "What time is it?",
+    fn: (argv, context) => {
+      //check for highnoon using Momentjs
+      if(Moment().format('Hm') == "1200") {
+        message = "It's High Noon!";
+      }else{
+        message = "It's " + Moment().format('LT');
```
Comment (SpyMaster356) [inline lib/commands/highnoon.js]: What's the LT format? Can you add a comment?

**Comment by step1041** [comment]
There, highnoon is now time, I added the new command to the changelog, and added some comments for MomentJS

**Pull request merged**